data/izam-mohammed.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "Izam Mohammed",
+    "type": "personal",
+    "displayStatsPublic": true,
+    "bio": "Open Source Enthusiast | **Data Scientist** | Freeelancer",
+    "avatar": "https://github.com/izam-mohammed.png",
+    "tags": [
+      "Open Source",
+      "Python",
+      "Data Science",
+      "Github",
+      "Machine Learning",
+      "Deep Learning"
+    ],
+    "socials": [
+      { "icon": "FaTwitter", "url": "https://twitter.com/izammohammed2" },
+      { "icon": "FaGithub", "url": "https://github.com/izam-mohammed" },
+      { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/izammohammed/" },
+      { "icon": "FaKaggle", "url": "https://www.kaggle.com/izammohammed" }
+    ],
+    "links": [
+      {
+        "group": "Socials",
+        "name": "Twitter: For updations of my day-to-day life",
+        "url": "https://twitter.com/izammohammed2",
+        "icon": "FaTwitter",
+        "color": "#00ACEE"
+      },
+      {
+        "group": "Socials",
+        "name": "GitHub: For my Open-source contributions and projects",
+        "url": "https://github.com/izam-mohammed",
+        "icon": "FaGithub"
+      },
+      {
+        "group": "Socials",
+        "name": "LinkedIn: Let's connect and share some ideas",
+        "url": "https://www.linkedin.com/in/izammohammed/",
+        "icon": "FaLinkedin"
+      },
+      {
+        "group": "Socials",
+        "name": "Kaggle: For my compititions and ML research",
+        "url": "https://www.kaggle.com/izammohammed",
+        "icon": "FaKaggle"
+      },
+      {
+        "group": "Socials",
+        "name": "Hashnode: For my blogs and articles",
+        "url": "https://hashnode.com/@izammohammed",
+        "icon": "SiHashnode"
+      },
+      {
+        "group": "Socials",
+        "name": "Medium: For my blogs in Medium",
+        "url": "https://medium.com/@izamdeveloper1",
+        "icon": "BsMedium"
```

Comment:
`BsMedium` is not yet supported by LinkFree, you can use any one of these. So update it accordingly. 

![image](https://user-images.githubusercontent.com/88786642/215284234-bd9ec7d0-d27f-4111-be33-d4313f55f551.png)